app/service/navigation_items_service.rb

```diff
@@ -19,6 +23,7 @@ def navigation_items
     navigation_items = []
     navigation_items << NavigationItem.new(text: I18n.t("header.mous"), href: mou_signatures_path, active: false) if should_show_mous_link?
     navigation_items << NavigationItem.new(text: I18n.t("header.users"), href: users_path, active: false) if should_show_user_profile_link?
```

Comment:
This is starting to feel rather busy and hard to read. 

Could you maybe refactor this method so it sort of follows the style we used here https://github.com/alphagov/forms-runner/blob/main/app/services/form_submission_service.rb#L133-L137 by making each item a private method with a guard to return nil if it shouldn't display  the nav. 

🤔  It feels like we should be using pundit policy to control whether or not a user should see the various links.

Reply:
Thanks this is loads better and easy to read. It is also easier to see what order each item would be in

Reply:
Yeah, good suggestion - it's a defintie improvement!